### PR TITLE
improve validation on intoto v0.0.2 type

### PR DESCRIPTION
**Pull request by bobcallaway**
This improves the validation of intoto v0.0.2 types when processed through the SearchLogQuery endpoint.

### CODE REVIEW

File: pkg/types/intoto/v0.0.2/intoto_v0_0_2_schema.json [hunk 1]
```diff
@@ -93,7 +93,10 @@
                     ],
                     "readOnly": true
                 }
-            }
+            },
+            "required": [
```
Comment (Hayden-IO) [inline pkg/types/intoto/v0.0.2/intoto_v0_0_2_schema.json:97]: To confirm, this shouldn’t break any existing callers, correct? The envelope should have always been specified?

**Review by Hayden-IO — APPROVED** [review]

**Comment by Hayden-IO** [comment]
The new codecov tests may need to be configured or disabled, they’re failing in other projects too.

**Comment by Hayden-IO** [comment]
https://docs.codecov.com/docs/commit-status to configure (this is actually a nice feature)

File: pkg/types/intoto/v0.0.2/intoto_v0_0_2_schema.json (hunk 1 above)
Comment (bobcallaway) [reply pkg/types/intoto/v0.0.2/intoto_v0_0_2_schema.json:97]: no, the envelope is required on upload anyway so this is actually more accurate of intent.

**Review by Hayden-IO — APPROVED** [review]

**Pull request merged**